platforms/os/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+alias(
```

Comment:
i don't see the value here for these aliases. why can't we just use them in the select statement?

Reply:
bzlmod isn't happy about that because the select statements ends up in the user's workspace which forces the user to dep on `platforms` in their MODULE.bazel

Reply:
oh, i see that makes sense. in that case can we move these into private?

Reply:
We could but they would then go in `npm/private` since they are being by `npm`; we may want to use them in `js/` the future so I figure `platforms` is a better home. Perhaps `platforms/private` tho we don't advertise `//platforms` as public API anywhere.

Reply:
I don't think this is a meaningful "public API surface" to bother about the private distinction.